### PR TITLE
follow up to the addition of AmpStoryCssTransformer

### PR DESCRIPTION
**Pull request by erwinmombay**
- add lts test
-  make sure optimizeAmpStory is off by default
- fix dvh-polyfill script
- add optimizeAmpStory to docs

### CODE REVIEW

**Comment by erwinmombay** [comment]
@sebastianbenz could you give some guidance on how to write an end to end test for this. I want to make sure the RewriteAmpUrls transformer doesn't conflict with how i'm writing the code here and doesn't do something like "lts/lts" in its path

**Comment by sebastianbenz** [comment]
You can add end-to-end tests in this directory: https://github.com/ampproject/amp-toolbox/tree/main/packages/optimizer/spec/end-to-end

1. Create a new directory with a story input file here: https://github.com/ampproject/amp-toolbox/tree/main/packages/optimizer/spec/end-to-end
1. Then run `npm run test:optimizer:snapshot` to generate the expected output
1. Manually inspect the output for any problems

**Comment by erwinmombay** [comment]
@sebastianbenz is there a way to similarly turn on an experiment flag in end-to-end tests like how i did it with lts as a comment or through a config.json?

**Comment by sebastianbenz** [comment]
e2e tests automatically run for default, lts and paired.

**Comment by sebastianbenz** [comment]
(paired doesn't make sense in the story sense, so it's safe to ignore)

**Comment by erwinmombay** [comment]
@sebastianbenz would it make sense for me to create a "experimental" spec where i turn on experimental flags?

**Comment by sebastianbenz** [comment]
How about creating a story specific configuration here: https://github.com/ampproject/amp-toolbox/blob/main/packages/optimizer/spec/end-to-end/EndToEndSpec.js#L33

**Comment by erwinmombay** [comment]
@sebastianbenz PTAL. (tests are failing because main is broken, needs https://github.com/ampproject/amp-toolbox/pull/1272 merged)

**Comment by erwinmombay** [comment]
@sebastianbenz tests are green now. Please take another look. Thanks!

File: packages/optimizer/README.md [hunk 1]
```diff
@@ -314,6 +315,16 @@ Enables a considerably faster scanning method in `amp-bind`, by injecting a `i-a
 - default: `true`
 - used by: [OptimizeAmpBind](lib/transformers/OptimizeAmpBind.js)
 
+### `optimizeAmpStory`
+
+Enabled server side rendering optimizations to improve amp-story performance such as linking to the amp-story-1.0.css, and server side rendering
```
Comment (sebastianbenz) [inline packages/optimizer/README.md:320]:
```suggestion
Enables AMP Story optimizations such as linking to the amp-story-1.0.css, and server side rendering
```

**Review by sebastianbenz — APPROVED** [review]

**Comment by sebastianbenz** [comment]
LGTM, one suggestion

**Comment by erwinmombay** [comment]
committed suggestion. should be green now and mergeable

**Comment by sebastianbenz** [comment]
Thanks Erwin!

**Pull request merged**